fawltydeps/settings.py

```diff
@@ -122,7 +122,8 @@ class Settings(BaseSettings):  # type: ignore
     code: Set[PathOrSpecial] = {Path(".")}
     deps: Set[Path] = {Path(".")}
     pyenv: Optional[Path] = None
-    custom_mapping: Optional[Path] = None
+    custom_mapping_file: Optional[Path] = None
```

Comment:
Same comment as above about using `custom_mapping_file` when `custom_mapping_path` is already in use elsewhere.

Reply:
I kept `custom_mapping_file` - we do not traverse directories here, and `_path` suggests we have either directory or a file.